common/autoware_lanelet2_utils/src/geometry.cpp

```diff
@@ -249,4 +251,35 @@ double get_lanelet_angle(
     segment.back().y() - segment.front().y(), segment.back().x() - segment.front().x());
 }
 
+geometry_msgs::msg::Pose get_closest_center_pose(
+  const lanelet::ConstLanelet & lanelet, const lanelet::BasicPoint3d & search_pt)
+{
+  lanelet::ConstLineString3d segment = get_closest_segment(lanelet.centerline(), search_pt);
+  if (segment.empty()) {
+    geometry_msgs::msg::Pose closest_pose;
+    closest_pose.position.x = lanelet.centerline().front().x();
```

Comment:
Let's use `to_ros` https://github.com/autowarefoundation/autoware_core/blob/031698c3e8f413186600bde5f26fc71a7f268a1b/common/autoware_lanelet2_utils/src/conversion.cpp#L117.
```suggestion
    closest_pose.position = to_ros(lanelet.centerline().front());
```